src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -40,7 +40,7 @@ def _get_client(service, parsed_args):
     account_name = parsed_args.account_name or os.getenv('AZURE_STORAGE_ACCOUNT')
     account_key = parsed_args.account_key or os.getenv('AZURE_STORAGE_KEY')
     connection_string = parsed_args.connection_string or os.getenv('AZURE_STORAGE_CONNECTION_STRING')
-    sas_token = parsed_args.sas_token or os.getenv('AZURE_SAS_TOKEN')
+    sas_token = parsed_args.sas_token or os.getenv('AZURE_STORAGE_SAS_TOKEN')
```

Comment:
I could see a user wanting to store this in the config file so the token was used in each new cmd window.  Is there a reason to not allow it to be read from the config file?

---
In reply to: [77391940](https://github.com/Azure/azure-cli/pull/804#discussion_r77391940) [](ancestors = 77391940)